### PR TITLE
AD LDS & ADAM do not have have `userAccountControl` attribute present.

### PR DESCRIPTION
**Pull request by affrae**
### Why:

We have a feature when using a LDAP service (and our documentation [discusses](https://docs.github.com/en/enterprise-server/admin/authentication/authenticating-users-for-your-github-enterprise-server-instance/using-ldap#enabling-ldap-sync) this) where you can use LDAP Sync with Active Directory (AD) to suspend and unsuspend users from GHES depending on the account status in AD.

However this feature only works if the `userAccountControl` attribute is present in the the Active Directory schema. For some flavours of AD, such as AD LDS / ADAM, this attribute is not present in a user record, instead the `msDS-UserAccountDisabled` provides the appropriate data to work from.

This is causing confusion and pain amongst customers who would like to use LDAP Sync and take our documentation to infer that all flavours of Active Directory are supported.

The PR is to update our LDAP Sync docs to clarify that AD LDS & ADAM do not have have `userAccountControl` attribute present.

### What's being changed:

Replace
```
{% note %}

**Note**: LDAP entries can only be marked as disabled if you use Active Directory and the `userAccountControl` attribute is present and flagged with `ACCOUNTDISABLE`.

{% endnote %}
```

with

```
{% note %}

**Note**: LDAP entries can only be marked as disabled if you use Active Directory and the `userAccountControl` attribute is present and flagged with `ACCOUNTDISABLE`. Some variations of Active Directory, such as AD LDS and ADAM, do not have the `userAccountControl` attribute present.

{% endnote %}
```

### Check off the following:

- [ ] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@affrae 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/admin/authentication/authenticating-users-for-your-github-enterprise-server-instance/using-ldap.md [hunk 1]
```diff
@@ -132,7 +132,7 @@ After you enable LDAP sync, a synchronization job will run at the specified time
 
 {% note %}
 
-**Note**: LDAP entries can only be marked as disabled if you use Active Directory and the `userAccountControl` attribute is present and flagged with `ACCOUNTDISABLE`.
+**Note**: LDAP entries can only be marked as disabled if you use Active Directory and the `userAccountControl` attribute is present and flagged with `ACCOUNTDISABLE`. Some variations of Active Directory, such as AD LDS and ADAM, do not have the `userAccountControl` attribute present.
```
Comment (vgrl) [inline content/admin/authentication/authenticating-users-for-your-github-enterprise-server-instance/using-ldap.md:135]: Thanks for this contribution @affrae !

I'll make one tiny change to the language and :shipit: 

```suggestion
**Note**: LDAP entries can only be marked as disabled if you use Active Directory and the `userAccountControl` attribute is present and flagged with `ACCOUNTDISABLE`. Some variations of Active Directory, such as AD LDS and ADAM, don't support the `userAccountControl` attribute.
```

**Review by vgrl — DISMISSED** [review]
I'll update the branch and get this merged once tests are passing 🍏

**Review by Naziran-hub — COMMENTED** [review]
PRIVACY POLICY

Vnly,Inc asks for and uses your personal information and is committed to keeping this information secure and protected. This Privacy Policy explains why we require personal information and how we collect, use, disclose and secure your personal information. This Privacy Policy applies to personal information we collect from you and from third parties both, online and offline but does not apply to any personal information collected by third parties from you.

By accessing or using the Website and Mobile Application, you signify that you have read, understood and agreed to be bound by this Privacy Policy. We reserve the right to update or modify this Privacy Policy at any time without prior notice. Therefore, we encourage you to review this Policy each time you access our Services on our Website and Mobile Application. We may also choose to notify you of material changes to the Privacy Policy by email or by means of a notice on our home page.

The following terms and definitions have been set out in this privacy policy:

“Vnly”, “We”, “Us”, and “Our” means Gimmicks Vnly (MY) Sdn. Bhd. which owns and operates the website, www.vnly.net (“Website and Mobile Application”)

“You” and “Your” means a user of the website and the products and services provided via the website and Mobile Application.

“Services” includes lending and any other related services by us via our Website and Mobile Application.

Collection of User Information:

To create an account on Vnly, you must provide us with the basic details and information required as part of our Customer Identification process and you agree to this Privacy Policy, which governs how we treat your information. During the use of our Services, Website and Mobile Application and App, we may collect information about you from you and from third parties. While some information must be mandatorily provided, others are optional and certain portions of the information will remain private and some will be shared with associated third parties. All information you provide to us is voluntary. This information is including but not limited to the following:

Information about you when you register or when you use our Website and Mobile Application.

Personal and financial information like your name, email address, contact number, gender, date of birth, contact information, address, etc.

Information about your transaction history like spend data, through our app, etc.

We may also collect other personal information such as SMS, contacts, location data, mobile device data, etc.

Your unique identifiers such as username and password, preferences information and transaction history.

Information collected through your Facebook, Twitter, LinkedIn or other social media accounts which have been linked to the account on the Website and Mobile Application.

Information that you provide when you write directly to us (including by email) or provide us over telephone.

Any other information that Vnly is required to collect as per specific mandate from any bank or as a requirement under applicable law.

Aggregate information and electronic data – like the pages viewed, how you navigate through the website and interact with web pages, etc.

By registering for a Vnly account or beginning an application on the Vnly website or mobile application for any product or service, you are consenting to be contacted by us by written notices, email messages, text messages, WhatsApp, or telephone, and, in connection with any such telephone calls, you consent to the use pre-recorded/artificial voice messages and/or automatic dialing devices, at any telephone number associated with your Vnly account, including mobile telephone numbers or at any email address or mailing address we have for you in our records or from other public and nonpublic databases we may lawfully access. You agree that Vnly may contact you using the information you provided with information and offers of services available through Vnly website and mobile application.

The information we collect about you will depend on the products and services we offer, on an ongoing basis. If you do not allow us to collect all the information we request, we may not be able to deliver all these services effectively.

You understand that, by creating an account or by registering through third-party sites, we and others will be able to identify you by your Vnly profile. However, we will not be liable under any circumstance for the generation of your profile or the information you provide therein. Vnly will also not be liable for any information collected in the form of SMS, contact, media or any other sources that the users might upload, the data collection process may extract, the use and misuse thereof.

All the information that you shall provide us is voluntary, including sensitive personal information. You understand that Vnly may use certain information of yours, which has been designated as ‘sensitive personal data or information’ under the IT Rules for the purpose of providing you with the Services and for sharing the information only with affiliates who are subject to this Privacy Policy, as will be explained further below.

Third-Party Sharing of Your Information

We will take utmost care to ensure that your personal information is not shared with the public and that your transactions on the Website and Mobile Application are confidential. If we use any third parties for any of these transactions, we will also make sure that confidentiality is maintained with these authorised third parties. We only share your personal information with affiliated third parties as permitted or required by law. When we share your personal information with vendors and service providers who perform functions on our behalf, we require that they keep your information confidential and secure and that they use it only as is reasonably necessary to carry out their work or comply with applicable law. We may share your personal information with affiliates and as part of joint marketing campaigns with other financial companies. In addition, personal information may be disclosed or transferred to our affiliates and as part of, or during negotiations of, a merger, consolidation, sale of company assets or equity, financing, acquisition, strategic alliance or in any other situation where personal information may be transferred as one of the business assets of Vnly.

Protecting Your Information

We maintain administrative, technical and physical safeguards designed to protect the security, confidentiality and integrity of your personal information. We store all the customer data on AWS cloud servers to reduce the risks of loss, misuse, unauthorised access, disclosure and alteration. Some of the safeguards we use are firewalls, SSL data encryption and access authorisation controls. In addition, your personal information resides on servers or in areas of facilities that only selected Vnly personnel and contractors have access to.

However, you understand and accept that there’s no guarantee that data transmission over the Internet will be completely secure and that any information that you transmit to us is at your own risk. We assume no liability for any disclosure of information due to errors in transmission, unauthorised third-party access to our Website and Mobile Application and databases or other acts of third parties, or acts or omissions beyond our reasonable control and you shall not be entitled to hold  Vnly responsible for any breach of security.

Your Choices Regarding Your Personal Information

Personal information that is not collected on the Website and Mobile Application or in connection with the Service is not generally subject to this Privacy Policy. The Website and Mobile Application may contain links to other websites which may collect personal information. For example, Vnly may have relationships with various service providers whose services you might find useful and are available to you from the Website and Mobile Application, from websites that are linked to the Website and Mobile Application, or in connection with the Service. Those providers may require you to provide personal information to use their services. Unless expressly provided to the contrary, the privacy policy applicable to that website, and not this Privacy Policy will govern the collection, use, disclosure and security of your personal information.

You can access all your personal and personally identifiable information that we maintain by logging in to your account and going to “My Profile”. This section of the site is password-protected to better safeguard your information. To protect the confidentiality of such personal information, you should not divulge your password to anyone. You are responsible for all uses of the Website and Mobile Application by any person using your customer ID and password. As a registered user, you can update physical address, phone numbers, education and employment information at any time on the website.

Tracking, Use of Cookies, Web Beacons and Similar Tracking Devices

Cookies are small data files that a website stores on your computer. Vnly uses cookies test page, afyet-sales, samesite for security and to personalise your web browsing experience. We may use cookies for some situations, such as

To allocate an identification number to your Internet browser (you cannot be identified from this number).

To make our Website and Mobile Application more user-friendly. Sent@vnly.net.

To determine if you have previously visited the website.

To track and report on website and online campaign performance.

To identify other pages or third-party websites you have accessed, help@vnly.net.

To detect fraud and for security purposes.

It is possible for you to configure your browser so that it does not receive cookies. However, this means you will be unable to use secure services that require cookies in order to participate. Some of those services include (but are not limited to) our online application and some member features.

Information from Advertisements on Other Websites
Hello@vnly.net.

We place and track advertisements on third-party websites. Like many advertisers, we advertise our products and services, and we contract with third parties to place advertisements, on websites not affiliated with us where we think the advertisements are likely to be most relevant. These advertisements may contain cookies or other mechanisms that allow tracking of your response to our ads. We and our contracted third parties track and report performance of our advertising and marketing campaigns by using cookies, web beacons, and other similar technology, which may be used to collect and store information about user visits, page visits and duration, and the specific ad or link that the user clicked on to visit the site. No personally identifiable information is stored during this process.

Contacting Vnly

If you have questions regarding our Privacy Policy or if you believe that Vnly has not complied with this Privacy Policy with respect to your personal information, please contact us at info@vnly.net. If you are reporting noncompliance with this Privacy Policy, in your email or letter, please describe in as much detail as possible (but without disclosing any sensitive information about you or third parties) the ways in which you believe that the Privacy Policy has not been complied with.


Legal Privacy 
Vnly Subprocessors
VnlyCode uses the subprocessors listed below to process customer personal information in connection with the Vnlycode Platform and Vnlycode Community. Veracode updates this subprocessor list periodically; the last update was made on September 15, 2021.  Vnlycode will notify you of planned updates to its subprocessor list through the Vnlycode Platform Legally and Vnlcode Community. If you have any objections to Vnlycode using a subprocessor please send an email to Privacy@Vnly.net stating your reason for the objection.  If Vnlycode does not receive a reasonable objection from you sent to Privacy@Vnly.net within 30 days of receiving notice of Vnlycode appointing a new subprocessor to this list, you will be deemed to have approved the use of such subprocessor.

 

Infrastructure & Security-Entity Name-Entity Type-Country

Amazon Web Services, Inc.-Cloud Service Provider
-United States, Germany

Okta, Inc.-User Authentication-United States, Germany

CoreSite-Data Center-United States

Pendo-User Experience-United States

PlexTrac-Penetration Testing Platform-United States

Splunk-Security-United States

 

Marketing & Sales-Entity Name-Entity Type-Country

Alyce-Marketing Services-United States

Avani Media-Marketing Services-United States

Conductor-Marketing Services-United States

Facebook-Marketing Services-United States

GearsCRM, Inc.-Marketing Services-United States

Geiger-Marketing Services-United States

Google-Marketing Services-United States

Lusha Systems, Inc.-Marketing Services-United States

LinkedIn-Marketing Services-United States

Marketo-Marketing Services-United States

Netsuite, Inc.-Sales Services-United States

Onclusive-Marketing Services-United States

OneTrust-Privacy Management Software-United States

Sales Navigator-Sales Services-United States

Salesforce-Sales & Marketing Services-United States

Tech Target, Inc.-Marketing Services-United States

Terminus-Marketing Services-United States

Twitter-Marketing Services-United States

ZoomInfo-Sales Services-United States

 
Customer Support-Entity Name-Entity Type-Country

Client Success, Inc.-Customer Support Services-United States

Accellion-Secure File transfer-United States

Skedulo-Customer Support Services-United States

 

Footer Menu
Legal-Privacy
Cloud Certifications

© 2021 VNLY, All Rights Reserved 33 Published Network Drive, Malaysia MY 22000

**Review by vgrl — APPROVED** [review]
👍

**Review by Naziran-hub — COMMENTED** [review]
# Contributor Covenant Code of Conduct

## Our Pledge

We as members, contributors, and leaders pledge to make participation in our
community a harassment-free experience for everyone, regardless of age, body
size, visible or invisible disability, ethnicity, sex characteristics, gender
identity and expression, level of experience, education, socio-economic status,
nationality, personal appearance, race, religion, or sexual identity
and orientation.

We pledge to act and interact in ways that contribute to an open, welcoming,
diverse, inclusive, and healthy community.

## Our Standards

Examples of behavior that contributes to a positive environment for our
community include:

* Demonstrating empathy and kindness toward other people
* Being respectful of differing opinions, viewpoints, and experiences
* Giving and gracefully accepting constructive feedback
* Accepting responsibility and apologizing to those affected by our mistakes,
  and learning from the experience
* Focusing on what is best not just for us as individuals, but for the
  overall community

Examples of unacceptable behavior include:

* The use of sexualized language or imagery, and sexual attention or
  advances of any kind
* Trolling, insulting or derogatory comments, and personal or political attacks
* Public or private harassment
* Publishing others' private information, such as a physical or email
  address, without their explicit permission
* Other conduct which could reasonably be considered inappropriate in a
  professional setting

## Enforcement Responsibilities

Community leaders are responsible for clarifying and enforcing our standards of
acceptable behavior and will take appropriate and fair corrective action in
response to any behavior that they deem inappropriate, threatening, offensive,
or harmful.

Community leaders have the right and responsibility to remove, edit, or reject
comments, commits, code, wiki edits, issues, and other contributions that are
not aligned to this Code of Conduct, and will communicate reasons for moderation
decisions when appropriate.

## Scope

This Code of Conduct applies within all community spaces, and also applies when
an individual is officially representing the community in public spaces.
Examples of representing our community include using an official e-mail address,
posting via an official social media account, or acting as an appointed
representative at an online or offline event.

## Enforcement

Instances of abusive, harassing, or otherwise unacceptable behavior may be
reported to the community leaders responsible for enforcement at
[INSERT CONTACT METHOD].
All complaints will be reviewed and investigated promptly and fairly.

All community leaders are obligated to respect the privacy and security of the
reporter of any incident.

## Enforcement Guidelines

Community leaders will follow these Community Impact Guidelines in determining
the consequences for any action they deem in violation of this Code of Conduct:

### 1. Correction

**Community Impact**: Use of inappropriate language or other behavior deemed
unprofessional or unwelcome in the community.

**Consequence**: A private, written warning from community leaders, providing
clarity around the nature of the violation and an explanation of why the
behavior was inappropriate. A public apology may be requested.

### 2. Warning

**Community Impact**: A violation through a single incident or series
of actions.

**Consequence**: A warning with consequences for continued behavior. No
interaction with the people involved, including unsolicited interaction with
those enforcing the Code of Conduct, for a specified period of time. This
includes avoiding interactions in community spaces as well as external channels
like social media. Violating these terms may lead to a temporary or
permanent ban.

### 3. Temporary Ban

**Community Impact**: A serious violation of community standards, including
sustained inappropriate behavior.

**Consequence**: A temporary ban from any sort of interaction or public
communication with the community for a specified period of time. No public or
private interaction with the people involved, including unsolicited interaction
with those enforcing the Code of Conduct, is allowed during this period.
Violating these terms may lead to a permanent ban.

### 4. Permanent Ban

**Community Impact**: Demonstrating a pattern of violation of community
standards, including sustained inappropriate behavior,  harassment of an
individual, or aggression toward or disparagement of classes of individuals.

**Consequence**: A permanent ban from any sort of public interaction within
the community.

## Attribution

This Code of Conduct is adapted from the [Contributor Covenant][homepage],
version 2.0, available at
[https://www.contributor-covenant.org/version/2/0/code_of_conduct.html][v2.0].

Community Impact Guidelines were inspired by 
[Mozilla's code of conduct enforcement ladder][Mozilla CoC].

For answers to common questions about this code of conduct, see the FAQ at
[https://www.contributor-covenant.org/faq][FAQ]. Translations are available 
at [https://www.contributor-covenant.org/translations][translations].

[homepage]: https://www.contributor-covenant.org
[v2.0]: https://www.contributor-covenant.org/version/2/0/code_of_conduct.html
[Mozilla CoC]: https://github.com/mozilla/diversity
[FAQ]: https://www.contributor-covenant.org/faq
[translations]: https://www.contributor-covenant.org/translations

**Comment by Naziran-hub** [comment]
# Contributor Covenant Code of Conduct

## Our Pledge

We as members, contributors, and leaders pledge to make participation in our
community a harassment-free experience for everyone, regardless of age, body
size, visible or invisible disability, ethnicity, sex characteristics, gender
identity and expression, level of experience, education, socio-economic status,
nationality, personal appearance, race, religion, or sexual identity
and orientation.

We pledge to act and interact in ways that contribute to an open, welcoming,
diverse, inclusive, and healthy community.

## Our Standards

Examples of behavior that contributes to a positive environment for our
community include:

* Demonstrating empathy and kindness toward other people
* Being respectful of differing opinions, viewpoints, and experiences
* Giving and gracefully accepting constructive feedback
* Accepting responsibility and apologizing to those affected by our mistakes,
  and learning from the experience
* Focusing on what is best not just for us as individuals, but for the
  overall community

Examples of unacceptable behavior include:

* The use of sexualized language or imagery, and sexual attention or
  advances of any kind
* Trolling, insulting or derogatory comments, and personal or political attacks
* Public or private harassment
* Publishing others' private information, such as a physical or email
  address, without their explicit permission
* Other conduct which could reasonably be considered inappropriate in a
  professional setting

## Enforcement Responsibilities

Community leaders are responsible for clarifying and enforcing our standards of
acceptable behavior and will take appropriate and fair corrective action in
response to any behavior that they deem inappropriate, threatening, offensive,
or harmful.

Community leaders have the right and responsibility to remove, edit, or reject
comments, commits, code, wiki edits, issues, and other contributions that are
not aligned to this Code of Conduct, and will communicate reasons for moderation
decisions when appropriate.

## Scope

This Code of Conduct applies within all community spaces, and also applies when
an individual is officially representing the community in public spaces.
Examples of representing our community include using an official e-mail address,
posting via an official social media account, or acting as an appointed
representative at an online or offline event.

## Enforcement

Instances of abusive, harassing, or otherwise unacceptable behavior may be
reported to the community leaders responsible for enforcement at
[INSERT CONTACT METHOD].
All complaints will be reviewed and investigated promptly and fairly.

All community leaders are obligated to respect the privacy and security of the
reporter of any incident.

## Enforcement Guidelines

Community leaders will follow these Community Impact Guidelines in determining
the consequences for any action they deem in violation of this Code of Conduct:

### 1. Correction

**Community Impact**: Use of inappropriate language or other behavior deemed
unprofessional or unwelcome in the community.

**Consequence**: A private, written warning from community leaders, providing
clarity around the nature of the violation and an explanation of why the
behavior was inappropriate. A public apology may be requested.

### 2. Warning

**Community Impact**: A violation through a single incident or series
of actions.

**Consequence**: A warning with consequences for continued behavior. No
interaction with the people involved, including unsolicited interaction with
those enforcing the Code of Conduct, for a specified period of time. This
includes avoiding interactions in community spaces as well as external channels
like social media. Violating these terms may lead to a temporary or
permanent ban.

### 3. Temporary Ban

**Community Impact**: A serious violation of community standards, including
sustained inappropriate behavior.

**Consequence**: A temporary ban from any sort of interaction or public
communication with the community for a specified period of time. No public or
private interaction with the people involved, including unsolicited interaction
with those enforcing the Code of Conduct, is allowed during this period.
Violating these terms may lead to a permanent ban.

### 4. Permanent Ban

**Community Impact**: Demonstrating a pattern of violation of community
standards, including sustained inappropriate behavior,  harassment of an
individual, or aggression toward or disparagement of classes of individuals.

**Consequence**: A permanent ban from any sort of public interaction within
the community.

## Attribution

This Code of Conduct is adapted from the [Contributor Covenant][homepage],
version 2.0, available at
[https://www.contributor-covenant.org/version/2/0/code_of_conduct.html][v2.0].

Community Impact Guidelines were inspired by 
[Mozilla's code of conduct enforcement ladder][Mozilla CoC].

For answers to common questions about this code of conduct, see the FAQ at
[https://www.contributor-covenant.org/faq][FAQ]. Translations are available 
at [https://www.contributor-covenant.org/translations][translations].

[homepage]: https://www.contributor-covenant.org
[v2.0]: https://www.contributor-covenant.org/version/2/0/code_of_conduct.html
[Mozilla CoC]: https://github.com/mozilla/diversity
[FAQ]: https://www.contributor-covenant.org/faq
[translations]: https://www.contributor-covenant.org/translations

File: content/admin/authentication/authenticating-users-for-your-github-enterprise-server-instance/using-ldap.md (hunk 1 above)
Comment (affrae) [reply content/admin/authentication/authenticating-users-for-your-github-enterprise-server-instance/using-ldap.md]: Sounds great - thank you!

**Pull request merged**